ml-data-hub-plugin/src/main/groovy/com/marklogic/gradle/DataHubPlugin.groovy

```diff
@@ -145,6 +145,10 @@ class DataHubPlugin implements Plugin<Project> {
                 "This consists of downloading user configuration files (entity models, entity model-based files, flows, and steps) as a zip, " +
                 "deleting all such user configuration files in the project directory, "+
                 "and finally extracting the contents of the downloaded zip into the project directory.")
+        project.task("hubCleanJobsData", type: CleanJobsDataTask, group: developGroup,
```

Comment:
The privilege is named "delete-jobs", and the role is "job-deleter", so the expected name for this task would naturally be "hubDeleteJobs". But - we already have such a task, which is specific to DHF 4. 

Now, we have "hubRunLegacyFlow" and also "hubExportLegacyJobs". I'd argue that a DHF 4 user making use of these tasks is already confused that 2 of the 4 "legacy" DHF 4 tasks have the word "Legacy" in them and the other 2 do not. And that's confusing for poor DHF engineers like me, that see "hubDeleteJobs" but have to realize that's actually DHF 4-specific. 

I think we can kill 2 birds with 1 stone here - let's rename hubDeleteJobs/hubImportJobs to hubDeleteLegacyJobs and hubImportLegacyJobs. We'll capture that in our release notes and update the docs - and the docs are already incorrect - https://docs.marklogic.com/datahub/5.4/tools/gradle/gradle-tasks.html - they show "hubExportJobs" instead of "hubExportLegacyJobs", and hubImportJobs isn't even listed. 

We can then call this new task what it should be and what users expect it to be - hubDeleteJobs. And the task is DeleteJobsTask.